ambry-api/src/main/java/com.github.ambry/config/FrontendConfig.java

```diff
@@ -44,6 +45,13 @@
   public final String frontendSecurityServiceFactory;
 
   /**
+   * The {@link AclServiceFactory} that needs to be used by AmbryBlobStorageService to
+   * make authorization decisions.
+   */
+  @Config("frontend.acl.service.factory")
```

Comment:
why do you need this? If the security service impl is going to create the acl service, there is no need for this here, correct?